ipwb/replay.py

```diff
@@ -684,6 +688,12 @@ def handler(signum, frame):
     # respWithLinkHeader = getLinkHeaderAbbreviatedTimeMap(path, datetime)
     # resp.headers['Link'] = respWithLinkHeader.replace('\n', ' ')
 
+    if status[0] == '3':
+        # Bad assumption that the URI-M will contain \d14 but works for now.
+        uriBeforeURIR = request.url[:re.search(r'/\d{14}/', request.url).end()]
+        newURIM = uriBeforeURIR + resp.headers['Location']
```

Comment:
This will probably fail in many cases. I think the simplistic assumption made here is that extracted `uriBeforeURIR` is going to be a directory URI and stored `Location` header will be a relative path. Both of these are unrealistic assumptions. I think you can simply return the `Location` header without any modification and let the SW deal with it. I am not sure how will it behave in case of a full URIs (probably it will be caught by the SW too).

Reply:
The location header prior to this is a URI-R. Without the prepending of the replay system, the redirect forwards the user to the live Web.

Reply:
uriBeforeURIR ends up being `http://localhost:5000/memento/12345678901234/`.

Reply:
I misunderstood the `uriBeforeURIR` earlier. However, `Location` header is not going to be a full URI all the time, it could very well be a relative or absolute path.